MVVM Example/MVVM Example/ListView/ListViewController.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  ListViewController.swift
+//  MVVM Example
+//
+//  Created by Niamh Power on 30/01/2018.
+//  Copyright © 2018 Novoda. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+class ListViewController: UIViewController {
+
+    fileprivate let viewModel: ListViewModel
+    fileprivate let tableView = UITableView(frame: .zero, style: .grouped)
+    fileprivate let dataSource = ListViewDataSource()
+
+    init(viewModel: ListViewModel) {
+        self.viewModel = viewModel
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupViews()
+        setupHierarchy()
+        setupLayout()
+        bindViewModel()
+    }
+
+    fileprivate func setupViews() {
+        tableView.dataSource = dataSource
+        tableView.delegate = self
+        setupDataSource()
```

Comment:
Maybe this method should be named differently, in `setupViews` I would imagine that you setup colours/fonts and not delegates and dataSource